edge-modules/load-gen/Settings.cs

```diff
@@ -100,10 +100,14 @@ public override string ToString()
                 { nameof(this.TestStartDelay), this.TestStartDelay.ToString() },
                 { nameof(this.TestDuration), this.TestDuration.ToString() },
                 { nameof(this.TrackingId), this.TrackingId },
-                { nameof(this.TransportType), Enum.GetName(typeof(TransportType), this.TransportType) },
-                { nameof(this.TestResultCoordinatorUrl), this.TestResultCoordinatorUrl.ToString() },
+                { nameof(this.TransportType), Enum.GetName(typeof(TransportType), this.TransportType) }
             };
 
+            this.TestResultCoordinatorUrl.ForEach((url) =>
```

Comment:
Option class overrided tostring method and it is good to use.  But I would like to make it consistent to use Uri as return type for TestResultCoordinatorUrl.

Reply:
I can make consistent in another PR